planetaryimage/image.py

```diff
@@ -93,6 +93,29 @@ def get_nested_dict(item, key_list):
         """Get value from nested dictionary using a key list. """
         return reduce(lambda x, y: x[y], key_list, item)
 
+    def save(self, file_to_write=None, overwrite=False):
+        if overwrite:
+            file_to_write = self.filename
+        serial_label = dumps(self.label)
+        label_sz = len(serial_label)
+        image_pointer = int(label_sz / self.label['RECORD_BYTES']) + 1
```

Comment:
Using ceiling function here. (int(x/y) + 1)
1. Using math.ceil breaks TRAVIS CI for Py26 and Py 27.
2. range() does't accept argument from np.ceil().